### DISCUSSION TITLE
Binding multiple REST API

### DISCUSSION THREAD

**Discussion by ludviggrenmo** [Budibase/budibase#5588] [Help]
Hi all. Im new to budibase and i would like to do a POC around combining multiple REST API.
Basically i want to create a table and iterate the id prop with another api to get multiple properties like displayName and so on. 

Has anyone done this, is it possible etc? Thanks for the help

**Comment by melohagan** [comment]
Hey @ludviggrenmo 

This is possible by chaining **Execute Query** actions: https://docs.budibase.com/docs/referencing

**Reply by ludviggrenmo** [reply]
I will look into this, thanks!
